concordia/admin.py

```diff
@@ -88,6 +91,24 @@ def unpublish_action(modeladmin, request, queryset):
 unpublish_action.short_description = "Unpublish selected"
 
 
+class ConcordiaUserAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("date_joined", "transcription_count")
```

Comment:
The `get_queryset()` override below adds `.annotate(Count("transcription"))` to the queryset which causes Django to add the `COUNT()` to the generated SQL and add it to each returned object. Since I didn't name it (e.g. `annotate(something=Count…)`) the default is `<field name>__<aggregation function name>` and then I just had the `transcription_count` method to return that property.